libs/blocks/card/cardUtils.js

```diff
@@ -35,8 +35,19 @@ export const addFooter = (links, container, merch) => {
     if (parentElement && document.body.contains(parentElement)) parentElement.remove();
   });
 };
-
 export const addWrapper = (el, section, cardType) => {
+  const observeClassListChanges = (classToRemove) => {
```

Comment:
this function is called from exactly one place in the code below and can be inlined for simplicity

Reply:
inlined